mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -27,12 +27,13 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 nested_query: !ruby/object:Api::Resource::NestedQuery
   keys:
     - monitoredProjects
-legacy_long_form_project: true
 description: "A [project being monitored](https://cloud.google.com/monitoring/settings/multiple-projects#create-multi) by a Metrics Scope."
 immutable: true
+error_retry_predicates: ['transport_tpg.IsMonitoringMetricsScopeLinkPermissionError']
```

Comment:
What is the reason to add this error retry predicates?

With this change, when getting the error `Error 403: The caller does not have permission`, `terraform apply` fails without retry.

Reply:
I wonder if this new logic is necessary for the patch release? If not, we can consider adding it later.

Reply:
Can you share your log? That error should be retried with this predicate.

Reply:
Sure. Please find the log: https://paste.googleplex.com/5286764091539456

Reply:
I've never seen this error before. What are the permissions on the project you're using?

Reply:
The error retry predicate is needed to match the DCL retry behavior when newly created projects are used. There's no way it could be causing the error you're seeing and the test is passing in VCR.

Reply:
I added the role `Monitoring Metrics Scopes Admin` and 
`Monitoring Metrics Scopes Viewer` to the GOOGLE_IMPERSONATE_SERVICE_ACCOUNT on the project `local-mediator-361721`

I don't think error retry predicate causing the error `The caller does not have permission`.  The way to handle the 403 error is changed with error retry predicate.  On google provider 4.72.0, it retries when seeing the error `The caller does not have permission`, but on this PR, it fails quickly without retry.

Reply:
The DCL doesn't check the details of the error before retrying. I could do that in this predicate but it would be dangerous because there's no way to set a custom timeout for the 403 error code.

Reply:
I run the test on this PR in teamcity and the test failed. 
https://ci-oss.hashicorp.engineering/viewLog.html?buildId=415685&tab=buildResultsDiv&buildTypeId=GoogleCloudBeta_ProviderGoogleCloudBetaMmUpstream

I guess the test passed before adding the error retry predicate and didn't run again after adding error retry predicate to the code.

Reply:
Can you try it again with this predicate that retries all 403 errors?

Reply:
Yeah, the test passed https://ci-oss.hashicorp.engineering/viewLog.html?buildId=415688&tab=buildResultsDiv&buildTypeId=GoogleCloudBeta_ProviderGoogleCloudBetaMmUpstream